lib/appdmg.js

```diff
@@ -387,7 +387,8 @@ module.exports = exports = function (options) {
     ds.setWindowSize(global.bkgsize[0], global.bkgsize[1])
 
     global.opts.contents.forEach(function (e) {
-      ds.setIconPos(path.basename(e.path), e.x, e.y)
+      var p = typeof e.title === 'undefined' ? path.basename(e.path) : e.title
+      ds.setIconPos(path.basename(p), e.x, e.y)
```

Comment:
Re: path.basename, whoops!

Re `/`, do you mean add a check to see if `title` is a valid filename?